.env.sample

```
@@ -0,0 +1,9 @@
+# MagicMirror² 환경변수 설정
+
+# API 서버 설정
+MM_API_BASE_URL=http://localhost:3000/api/v2
+MM_WS_BASE_URL=ws://localhost:3000/ws
+
+# 기타 설정
+MM_PORT=8080
+MM_FETCH_TIMEOUT=30000
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MM_FETCH_TIMEOUT key should go before the MM_PORT key

(UnorderedKey)